tests/test_TFNetworkLayer.py

```diff
@@ -1123,13 +1123,19 @@ def test_SplitDimsLayer_simple_time2():
 
 
 def test_SplitDimsLayer_resolve_dims():
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(3, -1)), (3, 5))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(3, 5)), (3, 5))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(5, -1)), (5, 3))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(-1, 3, 5)), (2, 3, 5))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, -1, 5)), (2, 3, 5))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, 3, -1)), (2, 3, 5))
-  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, 3, -1, 1)), (2, 3, 5, 1))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(3, -1), pad_to_multiples=False), (3, 5))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(3, 5), pad_to_multiples=False), (3, 5))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=3 * 5, new_dims=(5, -1), pad_to_multiples=False), (5, 3))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(-1, 3, 5), pad_to_multiples=False), (2, 3, 5))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, -1, 5), pad_to_multiples=False), (2, 3, 5))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, 3, -1), pad_to_multiples=False), (2, 3, 5))
+  assert_equal(SplitDimsLayer._resolve_dims(old_dim=2 * 3 * 5, new_dims=(2, 3, -1, 1), pad_to_multiples=False), (2, 3, 5, 1))
```

Comment:
Why did you change them? `pad_to_multiples=False` is anyway the default, right? Then better just leave it as-is.

Reply:
I set the default `pad_to_multiples=None` for `__init__` and `get_out_data_from_opts`, but not for `_resolve_dims` as that function does not know the actual input `Data` (thus we cannot infer it when the argument is `None`)
I can change it and set the default `pad_to_multiples=False` for `_resolve_dims`.

Reply:
I see. Yes, maybe that's better (default pad_to_multiples=False for _resolve_dims).